.github/workflows/docs-link.yml

```diff
@@ -1,14 +0,0 @@
-name: Docs Link
```

Comment:
do you know why this isn't displayed in the ci section?

Reply:
Let me have a look again...

Reply:
Reverting the change in this file as I might've found a cause... Let me give it a try in #12449.